bindings/python/examples/composite_example.py

```diff
@@ -0,0 +1,497 @@
+
+    indexation_key = 'Hello'
+    indexation_data = 'Tangle'
+    message = client.message(index=indexation_key, data_str=indexation_data)
+    print(message)
+
+    message_id = message['message_id']
+    print(f'Message id: {message_id}')
+
+    client.retry_until_included(message_id)
+    metadata = client.get_message_metadata(message_id)
+    print(f'Metadata: {metadata}')
+
+    milestone_index = metadata['milestone_index']
+    print(f'Milestone index: {milestone_index}')
+    if not (milestone_index is None):
+        milestone = client.get_milestone(milestone_index)
+        print(f'Message got referenced by milestone {milestone}')
+    else:
+        print('Message is not referenced by a milestone')
+
+
+def mnemonic():
+    node_url = "https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe"
+    client = iota_client.Client(nodes_name_password=[[node_url]])
+
+    mnemonic = client.generate_mnemonic()
+    print(f'Generated mnemonic: {mnemonic}')
+
+    seed = client.mnemonic_to_hex_seed(mnemonic)
+    addresses = client.get_addresses(
+        seed, input_range_begin=0, input_range_end=2)
+    print(f'List of generated public addresses: {addresses}')
+
+
+def multiple_outputs():
+
+    node_url = "https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe"
+    client = iota_client.Client(nodes_name_password=[[node_url]])
+
+    # Seed must contain non-zero balance
+    seed = os.getenv('MY_IOTA_SEED')
+    try:
+        message = client.message(seed,
+                                 outputs=[{'address': 'atoi1qpnrumvaex24dy0duulp4q07lpa00w20ze6jfd0xly422kdcjxzakzsz5kf',
+                                           'amount': 3_000_000},
+                                          {'address': 'atoi1qz4sfmp605vnj6fxt0sf0cwclffw5hpxjqkf6fthyd74r9nmmu337m3lwl2',
+                                           'amount': 2_800_000},
+                                          {'address': 'atoi1qzumqjtucwglfja746vvmr7n54ep88kcu2qvaquqrnx9qs2z8f4t6d7muyq',
+                                           'amount': 3_000_000}])
+        message_id = message['message_id']
+        print(
+            f'Transaction sent: https://explorer.iota.org/testnet/message/{message_id}')
+    except ValueError as e:
+        print(e)
+        print('Websites to send tokens: https://faucet.testnet.chrysalis2.com/ or https://faucet.tanglekit.de/')
+
+
+def peer():
+    node_url = "https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe"
+    client = iota_client.Client(nodes_name_password=[[node_url]])
+    try:
+        peers = client.get_peers()
+        print(f'Peers: {peers}')
+    except ValueError as e:
+        print(e)
+        print("You don't have the permission to get the peers.")
+
+
+def quorum():
+    node_1 = 'https://api.thin-hornet-0.h.chrysalis-devnet.iota.cafe'
+    node_2 = 'https://api.thin-hornet-1.h.chrysalis-devnet.iota.cafe'
+    node_3 = 'https://api.thin-hornet-2.h.chrysalis-devnet.iota.cafe'
```

Comment:
api.thin-hornet-2 doesn't exist
```suggestion
    node_3 = 'https://api.lb-0.h.chrysalis-devnet.iota.cafe'
```